repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/gatherinfo/GatherTestInformation.java

```diff
@@ -122,12 +125,12 @@ public FileVisitResult preVisitDirectory(Path file, BasicFileAttributes attrs) t
                         this.failingModulePath = failingModule.getCanonicalPath();
 
                         if (!this.skipSettingStatusInformation) {
-                            this.getInspector().getJobStatus().setFailingModulePath(this.failingModulePath);
-                            this.writeProperty("failingModule", this.failingModulePath);
-                            this.getInspector().getJobStatus().getMetrics4Bears().getTests().setFailingModule(this.failingModulePath);
+                            jobStatus.setFailingModulePath(this.failingModulePath);
```

Comment:
Actually we could even move the two lines below in the `setFailingModulePath` method.